src/docs/gold_standard_slot_examples.md

```diff
@@ -0,0 +1,444 @@
+# Gold Standard MIxS LinkML Slots Examples
+
+## Introduction
+
+This page lists a range of 'gold standard' examples of slots that are used in the MIxS LinkML schema.
+
+There are examples of slots that cover a range of common types of data, including:
+
+- Basic slots (i.e., match exactly to a LinkML `range` type):
+  - [Integer](#integer) (numeric)
+  - [Float](#float) (numeric)
+  - [String](#string) (free text)
+  - [Fixed list](#fixed-list) (enumeration)
+  - [Boolean](#boolean)
+- Complex (i.e., common more complex `range` types that are more specific to MIxS):
+  - [Structured text (text only)](#structured-text-text-only) (text only)
+  - [Structured text (with unit, measurement)](#structured-text-numeric-with-unit-measurement) (numeric with units)
+  - [Structured text (ontology term)](#structured-text-ontology-term)
+  - [URL](#url) (or other link utilising a URL like structure)
+  - [Mixed type (free text or ontology)](#mixed-type) (either free text, or from an ontology)
+
+You can use these as templates on how to construct your own new slot to the highest quality level expected.
+
+## Overview
+
+Every MIxS LinkML slot should have at a minimum the following attributes:
+
+- [`description`](https://linkml.io/linkml/schemas/metadata.html#providing-descriptions): the description of what the metadata term is for
+- [`title`](https://linkml.io/linkml-model/latest/docs/title/): a short human readable 'title' for the slot
+- [`examples`](https://linkml.io/linkml-model/latest/docs/examples/): examples values demonstrating how the slot should be used
+- [`in_subset`](https://linkml.io/linkml-model/latest/docs/in_subset/): the section of the schema that the slot belongs based on a [fix list of MIxS categories](https://github.com/GenomicsStandardsConsortium/mixs/blob/609b0f567486f64cb7061246588d8006f87fa138/src/mixs/schema/mixs.yaml#L21-L26)
+  - Note: this system may be replaced in the near future!
+- [`keywords`](https://linkml.io/linkml-model/latest/docs/keywords/): useful keywords to allow grouping of related slots together
+- [`slot_uri`](https://linkml.io/linkml-model/latest/docs/slot_uri/): a unique ID assigned by MIxS
+  - This likely only gets assigned upon acceptance and merging by the core GSC MIxS team
```

Comment:
```suggestion
  - This ID will be assigned upon acceptance and merging by the core GSC MIxS team.
```

Reply:
Note, we're not actually sure how the IDs are minted

Reply:
I should make a tool that reports the highest id so far. I wonder if @sujaypatil96 could help me expose that on the web pages.

Reply:
I guess leaving it to the ultimate repsonsbility of the core MIxS team is then fine (they can always delegate)